_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+A key concept in this debate is the term of the exchange. For a good to be money it is implicit that it serves well its purpose in the short term, which is also implicit in the words "most salable" within Menger's definition of money "the **most salable** of commodities", which [Carlos Bondone](http://www.carlosbondone.com/en/theory-of-economic-time/economic-theory/development-of-concepts.html) has generalized to "the most salable economic good". If it only serves its purpose in longer terms it might not be money but, it would still be a medium of indirect exchange.
+
+But, in this post, we are interested in the bootstrapping process to achieve that salability.  Let's refer back to the stage of goods being just a medium of indirect exchange. I stated above that I would rely on cereal's utility as food to foresee its future demand because the concept of medium of indirect exchange would still be an unknown abstraction for me. If we fast forward to the context where humans already grasp the concept of medium of exchange, why shouldn't they be able to foresee the monetary demand of any good regardless if it has other uses or not? That is to say: easy to transport, divisible, durable, difficult to fake, etc. This is clearly the case for Szabo's "collectibles" and also Bitcoin's case.  From a historical point of view, I find it rather unlikely that humans used otherwise useless goods as a medium of exchange without first having discovered the medium of exchange concept through other consumptive goods, but it is certainly theoretically possible.
+
+In the same way that discovering the concepts of adding and subtracting quantities is more natural through comparing physical quantities (i.e. beads, as in an abacus), that does not necessarily prevent humans to skip that step and directly invent written numerals and ledgers.  Or, if for example, the standard within the abacus was the sexagesimal numeral system, that does not mean that is mandatory to go through an abacus stage of the decimal system in order to be able to adopt the written representation of the decimal numeral system. We must not confuse the supporting medium by which we historically experiment and discover a concept (i.e. beads or commodities) with the concept itself (arithmetics or Money).
+
+If something arises (bitcoin) that can fulfill the sole use case of a medium of exchange better than the incumbents (barter, salt, gold, fiat etc.), then that is where the demand can be found. Can this demand can be speculative? Yes, it is speculation on the possibility of the good becoming money in the future.
+
+Regarding completely new monetary goods and the fact that they don't have any other use than being a medium of exchange, Mises had the following concern (_Human Action_ - Chapter 17 section 4 – "The determination of purchasing power of money"):
```

Comment:
Reword to "Mises had the following concern regarding new monetary goods that have no other use than a medium of exchange....